datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -17,11 +17,16 @@ import {
     Name,
     TitleRole,
     Team,
+    LocationSection,
+    LocationSectionText,
+    UserDetails
 } from '../shared/SidebarStyledComponents';
 import EntityGroups from '../shared/EntityGroups';
 import { mapRoleIcon } from '../../identity/user/UserUtils';
 import { useUserContext } from '../../context/useUserContext';
 import { useBrowserTitle } from '../../shared/BrowserTabTitleContext';
+import GlobeIcon from '../../../images/Globe.svg';
+import { countries } from 'country-data-list';
```

Comment:
Absolute importts should go at the top of the file!

Linter is probably already complaining about this.